### ISSUE TITLE
raw binary not working on centos 7

### ISSUE THREAD

**Issue by arcanosam** [brave/browser-laptop#13901]
I download the binary version from https://laptop-updates.brave.com/latest/linux64.

Run it, the process start, but the browser not open.

Then I try this commands (I do it the same commands on google chrome, when I download the rpm and extract only the folder with google chrome software

    sudo chown root:root -R ~/Tools/Brave/
    sudo chmod 755 -R ~/Tools/Brave/
    sudo chmod 4755 -R ~/Tools/Brave/brave

Not work either.

Some suggestion that I could generate a log to see what the real error is?
Or how I fix this?

thanks guys. I am appreciate very much the browser

**Comment by hugobuddel** [comment]
Perhaps this is because `--no-sandbox` is necessary on Centos?

Or namespaces need to be turned on: https://github.com/brave/browser-laptop/issues/6902#issuecomment-375712955

But there should be a message on the terminal when starting Brave.

**Comment by arcanosam** [comment]
Hi. I don't tried to run on terminal because I made a shortcut icon and I read somewhere that to run without --no-sandbox it's not good.

So I gived it  a try. Running without on command line. This is the error:

      [3177:3177:0425/083605.343049:ERROR:zygote_host_impl_linux.cc(90)] 
      Running as root without --no-sandbox is not supported. See https://crbug.com/638180.

But when a I run it with --no-sandbox, happens the same: starts the process, the icon appear show that is processing, but browser not appear (this when I click my shortcut icon)

When I try it on terminal running with this parameter that's the error:

    (process:3222): Gtk-WARNING **: This process is currently running setuid or setgid.
    This is not a supported use of GTK+. You must create a helper
    program instead. For further details, see:

    http://www.gtk.org/setuid.html

    Refusing to initialize GTK+.

The link  about namespaces you pass, I tried it to use the simple solution that is edit max_user_namespaces and set a value like sayed: 10000

But I can't edit, the message was "permission denied". 

I try to figure out later what can I do, because even with sudo I can't execute the command ou by vim

Thanks for the help and the tips. I think this could be works. 

As soon I try make this namespaces solution works, I'm back here to tell what happened.

**Comment by hugobuddel** [comment]
Good luck @arcanosam, you can probably figure something out.

Perhaps you can try to start with a clean slate, that is, by deleting Brave and unzipping the tarball again. Some of the errors you are currently getting seem to be due to the `chmod 4755` you did earlier, so you risk mixing different solutions together such that they don't work anymore.

There seem to be three ways to make it work (I'm no expert here):

- Just use `--no-sandbox`. That's what I'm doing.
- Fix the user namespaces. Several ways of doing so are described in #6902, not sure which one you'd need.
- Fix it through setuid. This seems to be what you were trying; I have no idea how that works.

**Comment by arcanosam** [comment]
yeah... I think I have must apologies.

When I tryed at first, I think Brave was executed without --no-sandbox parameter. That's the reason not worked at first moment.

After this, I maked the permission changes and not worked either, but I tryed now by command line prefixing sudo and Brave runed perfectly.

So, my problem really was not using --no-sandbox on my short icon
and after that the change of permissions.

sorry guys and @hugobuddel 

thanks for your clarifications.
